### PR TITLE
Remove instance blacklist

### PR DESCRIPTION
**Pull request by alexgleason**
Since today Tusky was banned from Google Play, and you now understand the harmful effects of censorship, this MR removes the instance blacklist in Tusky.

1. **gab.com** is unblacklisted. It no longer exists as a Fediverse implementation and is not even compatible with this app anymore.
2. **spinster.xyz** is unblacklisted. It's time to stop censoring feminists.

**Censorship harms marginalized people the most.** We are not your enemy. Our enemy is monopolistic tech corporations.

### CODE REVIEW

**Comment by seanking2919** [comment]
Absolutely. We as the fediverse should be focusing on providing a good decentralized alternative to the centralized monopolistic tech corporations. Not censoring and dividing each other based on differences of opinion. As Noam Chomsky once said, "If you’re really in favor of free speech, then you’re in favor of freedom of speech for precisely the views you despise. Otherwise, you’re not in favor of free speech."

**Comment by GatoOscuro** [comment]
**I agree that by default NO instance, pod, server and so on should be censored. That is at the option of each administrator of "x" service, but by default should not close the doors because it is like raising the flag of "freedom of speech" and include minorities, include sub-groups and so on.

NO to generalized blocking.**

**Comment by ChanoSan** [comment]
Agreed. Instance blocks are best left up to individual instance moderators, not shoehorned in by the developers of what's supposed to be a general-use application.

**Comment by M-i-k-o-t-o** [comment]
> spinster.xyz is unblacklisted. It's time to stop censoring feminists.

Weren't you censoring people against the TERF ideology in that instance? :blobcatthink: Kinda ironic for you to demand this.

Also I am pretty sure that they are not censored "just" for being feminists, but rather because it is a TERF instance. It is disingenuous of you to try and pretend that it is just a feminist instance and that this is why it was censored.

**Comment by icyphox** [comment]
Regardless of what the instance is about, let the users decide for
themselves. Censoring instances that you disagree with politically (or
for whatever other reason), in the app, is unacceptable. Didn't the
author himself claim, and I quote:

> They don't seem to understand that one can view any content with Tusky
> and that it is not possible for the app developers to check any of it.
> https://chaos.social/@ConnyDuck/105904015276457450

Which is patently false.

--
Anirudh Oppiliappan
https://icyphox.sh

**Comment by ralsei** [comment]
The enemy is monopolistic tech corporations, sure. The enemy also isn't trans people.

**Comment by alexgleason** [comment]
image

Since I'm being accused of reporting Tusky and can't defend myself because I'm blocked, I want to make something very clear:

I'm *against censorship*. This is the whole point. I don't report people to corporations I hate to try to ruin them. This is what you do, not what I do. I'm *against* Google arbitrarily removing apps, including yours.

I'm building my own solutions to problems, not trying to sabotage yours. I don't even have a Google account.

You still don't get it.

You're lucky that Google reinstated your app. They didn't have the same decency to even respond when it happened to us. Google is not on our side.

**Comment by seanking2919** [comment]
> 
> 
> > spinster.xyz is unblacklisted. It's time to stop censoring feminists.
> 
> Weren't you censoring people against the TERF ideology in that instance? :blobcatthink: Kinda ironic for you to demand this.
> 
> Also I am pretty sure that they are not censored "just" for being feminists, but rather because it is a TERF instance. It is disingenuous of you to try and pretend that it is just a feminist instance and that this is why it was censored.

And even if it is a TERF instance and as much as I despise TERFs such as J.K. Rowling, it's pointless to make the arbitrary decision to block people from using that instance on something that's supposed to be just a client for interacting with fediverse-compatible instances. The author of Tusky himself said:

> They don't seem to understand that one can view any content with Tusky and that it is not possible for the app developers to check any of it.
> https://chaos.social/@ConnyDuck/105904015276457450

And hard-coding blacklisting of instances into Tusky contradicts that point. You're either a fediverse client or you're not. Decide what you want to be.

**Comment by dev-kittens** [comment]
my 2 cents: i strongly disagree with a lot of @alexgleason's ideas, and i can choose to block his posts if something he says upsets me; or if something on spinster.xyz upsets me (which it often does) i can block the domain from my instance.

however, that's *my* choice as someone who handles content; i don't expect to be able to tell other people what content they should be able to access, especially if they are running software on their own devices - and especially if said software is open-source, and they would be able to modify it to access content that i disagree with.

in short, i don't think Tusky (or any other open-source project, for that matter) should attempt to limit the content that can be accessed through it. it benefits nobody, and without going against the spirit of open-source, it's quite useless

**Comment by seanking2919** [comment]
> 
> 
> my 2 cents: i strongly disagree with a lot of @alexgleason's ideas, and i can choose to block his posts if something he says upsets me; or if something on spinster.xyz upsets me (which it often does) i can block the domain from my instance.
> 
> however, that's _my_ choice as someone who handles content; i don't expect to be able to tell other people what content they should be able to access, especially if they are running software on their own devices - and especially if said software is open-source, and they would be able to modify it to access content that i disagree with.
> 
> in short, i don't think Tusky (or any other open-source project, for that matter) should attempt to limit the content that can be accessed through it. it benefits nobody, and without going against the spirit of open-source, it's quite useless

Absolutely agreed. I don't think any open-source project should to limit the content that can be accessed through it.
